devices/roku-tv.groovy

```diff
@@ -110,7 +112,7 @@ private def parseInstalledApps(Node body) {
     }
     
     body.app.each{ node ->
-        if (node.attributes().type != "appl") {
+        if (node.attributes().type != "appl" && (!includeInputChildDevices || node.attributes().type != "tvin")) {
```

Comment:
What TV do you have?  On mine, the channel app is simply "Antenna TV".  What is "tvin" ?  maybe this is what I have too, and I am just not seeing it because I haven't looked at the raw JSON in a while.

Reply:
TCL 4 series, this change made mine work. Might be different for different brands? ¯\_(ツ)_/¯